src/Diagnostics.RuntimeHost/Controllers/ProcessHealthController.cs

```diff
@@ -13,18 +15,44 @@ public class ProcessHealthController : Controller
     {
         private ISourceWatcherService _sourceWatcherService;
         private ICompilerHostClient _compilerHostClient;
+        private IConfiguration _configuration;
 
-        public ProcessHealthController(ISourceWatcherService sourceWatcherService, ICompilerHostClient compilerHostClient)
+        public ProcessHealthController(ISourceWatcherService sourceWatcherService, ICompilerHostClient compilerHostClient, IConfiguration Configuration)
         {
             // These dependencies are injected for the services to start.
             _sourceWatcherService = sourceWatcherService;
             _compilerHostClient = compilerHostClient;
+            _configuration = Configuration;
+        }
+
+        public static bool PingExternal(string pingUrl)
```

Comment:
this can be a private method instead of public

Reply:
My bad.